command/alloc_status.go

```diff
@@ -689,3 +691,77 @@ func (c *AllocStatusCommand) sortedTaskStateIterator(m map[string]*api.TaskState
 	close(output)
 	return output
 }
+
+func (c *AllocStatusCommand) outputTaskVolumes(alloc *api.Allocation, taskName string, verbose bool) {
+	var task *api.Task
+	var tg *api.TaskGroup
+FOUND:
+	for _, tg = range alloc.Job.TaskGroups {
+		for _, task = range tg.Tasks {
+			if task.Name == taskName {
+				break FOUND
+			}
+		}
+	}
+	if task == nil || tg == nil {
+		c.Ui.Error(fmt.Sprintf("Could not find task data for %q", taskName))
+		return
+	}
+	if len(task.VolumeMounts) == 0 {
+		return
+	}
+	client, err := c.Meta.Client()
+	if err != nil {
+		c.Ui.Error(fmt.Sprintf("Error initializing client: %s", err))
+		return
+	}
+
+	var hostVolumesOutput []string
+	var csiVolumesOutput []string
+	hostVolumesOutput = append(hostVolumesOutput, "ID|Read Only")
+	if verbose {
+		csiVolumesOutput = append(csiVolumesOutput,
+			"ID|Plugin|Provider|Schedulable|Mount Options")
+	} else {
+		csiVolumesOutput = append(csiVolumesOutput, "ID")
```

Comment:
Do we want `AccessMode` here, to match host volume's `Read Only`? I think we probably do want it in the verbose output

Reply:
I double-checked and we actually have `Read Only` from the `volume_mount` data we could use here, and it's more accurate than the data we'd get from `client.CSIVolumes().Info` anyways because the operator can mount a r/w volume as read-only for this task. So I'll add that for the non-verbose output.

Reply:
Done in 51a056f. Going to wait for tests and then merge this.